docs/source/specs/openapi.json

```diff
@@ -1846,6 +1846,100 @@
           }
         }
       }
+    },
+    "/permissions/options/": {
+      "get": {
+        "tags": [
+          "Permission"
+        ],
+        "summary": "List the available options for fields of permissions for a tenant",
+        "description": "By default, options of application is returned. And could be resource_type or verb on demand.",
+        "operationId": "listPermissionOptions",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/QueryLimit"
+          },
+          {
+            "$ref": "#/components/parameters/QueryOffset"
+          },
+          {
+            "name": "field",
+            "in": "query",
+            "description": "specify which fields of permission to display",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "enum": [
+                "application",
+                "resource_type",
+                "verb"
+              ]
+            }
+          },
+          {
+            "name": "application",
+            "in": "query",
+            "description": "Exact match for the application name of a permission. You may also use a comma-separated list to match on multiple applications.",
```

Comment:
I don't think these descriptions are accurate for `application`, `resource_type` and `verb` here? It's not the match of the app name, it's the option filter key? Also doesn't support multiple values? Assuming it's just because it's a copy/paste from the permission entry? 😄

Reply:
Yea, good catch! I changed that description. But it does support multiple values 😁 I just added a unit test to ensure that "test_return_options_with_comma_separated_filter"